### PR TITLE
 Add robb edge drivers ROB_200-004-0, ROB_200-011-0, ROB_200-011-0

### PR DESCRIPTION
**Pull request by zwave-mke**
@lelandblue sorry for the circumstances. We have cleaned up the branch and will move the other drivers to another branch to keep them clean.

### CODE REVIEW

File: drivers/SmartThings/zigbee-button/fingerprints.yml [hunk 1]
```diff
@@ -138,4 +138,4 @@ zigbeeManufacturer:
     deviceLabel: eWeLink Button
     manufacturer: eWeLink
     model: SNZB-01P
-    deviceProfileName: one-button-battery
+    deviceProfileName: one-button-battery
```
Comment (dljsjr) [inline drivers/SmartThings/zigbee-button/fingerprints.yml:141]: nit: No newline at end of file.

File: drivers/SmartThings/zigbee-switch/profiles/light-level-power-energy.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  - id: powerMeter
+    version: 1
+  - id: energyMeter
+    version: 1
+  - id: firmwareUpdate
+    version: 1
+  - id: refresh
+    version: 1
+  categories:
+  - name: Light
```
Comment (dljsjr) [inline drivers/SmartThings/zigbee-switch/profiles/light-level-power-energy.yml:22]: nit: No newline at end of file

File: drivers/SmartThings/zigbee-button/src/zigbee-multi-button/init.lua [hunk 3]
```diff
@@ -14,6 +14,8 @@
 
 local capabilities = require "st.capabilities"
 local supported_values = require "zigbee-multi-button.supported_values"
+local log = require "log"
```
Comment (dljsjr) [inline drivers/SmartThings/zigbee-button/src/zigbee-multi-button/init.lua:17]: Both of these `requires` are unused, and should be removed.

**Review by dljsjr — APPROVED** [review]
Just a few style comments. LGTM otherwise.

File: drivers/SmartThings/zigbee-switch/src/robb/init.lua [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      return true
+    end
+  end
+  return false
+end
+
+local function energy_meter_handler(driver, device, value, zb_rx)
+  local raw_value = value.value
+  local multiplier = device:get_field(constants.SIMPLE_METERING_MULTIPLIER_KEY) or 1
+  local divisor = device:get_field(constants.SIMPLE_METERING_DIVISOR_KEY) or 1000000
```
Comment (greens) [inline drivers/SmartThings/zigbee-switch/src/robb/init.lua:40]: you could just change your `added` method to set this field to 1000000 and avoid repeating the defaults here

File: drivers/SmartThings/zigbee-switch/src/robb/init.lua [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+end
+
+local function power_meter_handler(driver, device, value, zb_rx)
+  local raw_value = value.value
+  local multiplier = device:get_field(constants.ELECTRICAL_MEASUREMENT_MULTIPLIER_KEY) or 1
+  local divisor = device:get_field(constants.ELECTRICAL_MEASUREMENT_DIVISOR_KEY) or 10
+
+  raw_value = raw_value * multiplier / divisor
+  device:emit_event(capabilities.powerMeter.power({ value = raw_value, unit = "W" }))
+end
```
Comment (greens) [inline drivers/SmartThings/zigbee-switch/src/robb/init.lua:53]: you use the same unit for both of the attributes handled here? In our defaults I think we expect mW from instantaneous demand

File: drivers/SmartThings/zigbee-button/fingerprints.yml (hunk 1 above)
Comment (zwave-mke) [reply drivers/SmartThings/zigbee-button/fingerprints.yml:141]: changed from CRLF to LF

File: drivers/SmartThings/zigbee-switch/profiles/light-level-power-energy.yml (hunk 2 above)
Comment (zwave-mke) [reply drivers/SmartThings/zigbee-switch/profiles/light-level-power-energy.yml:22]: changed from CRLF to LF

File: drivers/SmartThings/zigbee-button/src/zigbee-multi-button/init.lua (hunk 3 above)
Comment (zwave-mke) [reply drivers/SmartThings/zigbee-button/src/zigbee-multi-button/init.lua]: done

File: drivers/SmartThings/zigbee-switch/src/robb/init.lua (hunk 4 above)
Comment (zwave-mke) [reply drivers/SmartThings/zigbee-switch/src/robb/init.lua]: simplified, cause it's not really needed.

File: drivers/SmartThings/zigbee-switch/src/robb/init.lua (hunk 5 above)
Comment (zwave-mke) [reply drivers/SmartThings/zigbee-switch/src/robb/init.lua]: simplified too

**Review by greens — APPROVED** [review]

**Comment by zwave-mke** [comment]
@greens commits squashed. Sorry for the delay ...

**Comment by zwave-mke** [comment]
@greens commits squashed. Sorry for the delay ...

**Comment by zwave-mke** [comment]
@greens commits squashed. Sorry for the delay ...

**Comment by greens** [comment]
@zwave-mke I think we're waiting on UL to get back to us before the merge, but thanks for keeping this PR nice and tidy.

**Pull request merged**